### PR TITLE
[Behat] [DX] Table row counter on index page

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Page/Admin/Crud/IndexPage.php [hunk 1]
```diff
@@ -76,6 +76,14 @@ public function isResourceOnPage(array $parameters)
     }
 
     /**
+     * @return int
+     */
+    public function countTableRecords()
```
Comment (pjedrzejewski) [inline src/Sylius/Behat/Page/Admin/Crud/IndexPage.php]: We should not assume that it is a table. Index page contains a collection (paginated or not) of items, so we should have something like ``countItems``?

Comment (pjedrzejewski) [inline src/Sylius/Behat/Page/Admin/Crud/IndexPage.php]: I meant that on the interface, we of course can use table accessor in our index page implementation, but the interface should not assume that it is a table.

Comment (lchrusciel) [inline src/Sylius/Behat/Page/Admin/Crud/IndexPage.php]: 👍

**Comment by michalmarcinkowski** [comment]
Looks good to me :)

**Comment by pjedrzejewski** [comment]
Thank you Łukasz!

**Pull request merged**